src/k-provider/ott/services/bookmark/bookmark-service.js

```diff
@@ -39,6 +47,7 @@ export default class OTTBookmarkService extends OTTService {
       playerData: playerData
     };
     if (bookmark.programId) bookmarkServiceParams.programId = bookmark.programId;
+    this._logger.debug('the processed bookmark', bookmarkServiceParams);
```

Comment:
what does it mean? let's have more meaningful debug output.
For example: `bookmark added ${bookmarkServiceParams }`